test/fdo-postgres.sh

```diff
@@ -160,7 +160,9 @@ sudo podman run \
     --privileged \
     -v "$PWD"/export-ov:/export-ov:z \
     localhost/clients \
-    fdo-owner-tool export-manufacturer-vouchers postgres "postgresql://${POSTGRES_USERNAME}:${POSTGRES_PASSWORD}@${POSTGRES_IP}/${POSTGRES_DB}" /export-ov/ | grep "exported"
+    fdo-owner-tool export-manufacturer-vouchers "http://${FDO_MANUFACTURING_ADDRESS}:8080" --path /export-ov | grep "exported"
```

Comment:
Don't we return the corresponding exit code? I mean, why don't we check the exit code instead of the program output?

Reply:
that's a good question - I just adapted to the new command w/o changing the logic

Reply:
Will address this in a follow up!

Reply:
See https://github.com/fdo-rs/fido-device-onboard-rs/issues/680